config/environments/production.rb

```diff
@@ -23,6 +23,10 @@
   # Disable serving static files from `public/`, relying on NGINX/Apache to do so instead.
   # config.public_file_server.enabled = false
 
+  # Store files on Amazon S3.
+  config.active_storage.service = :amazon
```

Comment:
You're gonna configure Amazon in production??

Reply:
According to [Active Storage](https://edgeguides.rubyonrails.org/active_storage_overview.html) it is configured there. 
Or what do u mean?

Reply:
Take into consideration that when you deploy you need to ctually configure the amazon s3 bucket